tcii18n/translator.py

```diff
@@ -0,0 +1,13 @@
+import csv
+
+
+class Translator:
+    def __init__(self, filename):
+        self.reader = csv.reader(open(filename, 'r'))
```

Comment:
The first solution was opening a file on each call to a translation.
This new version is opening a file only when creating a new instance.
When using it, we probably only want to call the instance once and then call the `translate()` as often as required.